libs/cardano-ledger-core/src/Cardano/Ledger/UMap.hs

```diff
@@ -240,7 +243,7 @@ instance Crypto c => DecShareCBOR (UMElem c) where
     decodeRecordNamed "UMElem" (const 4) $
       UMElem
         <$> decCBOR
-        <*> decCBOR
+        <*> ifDecoderVersionAtLeast (natVersion @9) (mempty <$ dropCBOR (Proxy @(Set Ptr))) decCBOR
```

Comment:
I guess we're not doing the equivalent thing for EncCBOR  (encCBOR Set.empty) because the map is empty anyway, right?

Reply:
Correct.